app/javascript/src/NewService/components/FormElements/ServiceManualListItems.tsx

```diff
@@ -35,9 +36,12 @@ const ServiceManualListItems: FunctionComponent<Props> = ({
           label="System name"
         />
         <input id="service_system_name" maxLength={255} name="service[system_name]" type="text" value={systemName} onChange={onChange(setSystemName)} />
-        <p className={`${errors.system_name ? 'inline-errors' : 'inline-hints'}`}>
-          Only ASCII letters, numbers, dashes and underscores are allowed.
-        </p>
+        <HelperText>
+          <HelperTextItem>
+            <p className={`${errors.system_name ? 'inline-errors' : 'inline-hints'}`}> { errors.system_name ? errors.system_name : '' } </p>
```

Comment:
@jlledom  this syntax is not a valid syntax in tsx file

 
also inline hint is necessary, because when error is not shown a text is shown in greyish color  like this 

![image](https://user-images.githubusercontent.com/84672731/232989209-f6dc28b7-9ec8-4ee2-a350-ed2325e7daf8.png)

so that cannot be removed.

if looks good then can we merge this now???

Reply:
The hint is shown always, no matter if there's error or not, see the [line 42](https://github.com/3scale/porta/pull/3287/files#diff-37e20e23cff9e24a4bdc0d10b49e347c079a9f0124106a0772e7fad4152ed1a8R42). Therefore, `inline-hints` is not required in the line 41.

This could probably work:
```suggestion
            <p className='inline-errors'> { errors.system_name ? errors.system_name : '' } </p>
```